drivers/overlay/ov_serf.go

```diff
@@ -40,7 +40,7 @@ func (d *driver) serfInit() error {
 
 	config := serf.DefaultConfig()
 	config.Init()
-	config.MemberlistConfig.BindAddr = d.bindAddress
+	config.MemberlistConfig.BindAddr = d.advertiseAddress
```

Comment:
Yes. it is. Thats because ov_serf is not used in swarm-mode. It is used ONLY for the non-swarm-mode. In the non-swarm-mode, advertiseAddress is always the bindAddress and hence the above code is correct.